examples/rsp-next-ts-17/styles/globals.css

```diff
@@ -11,6 +11,3 @@ a {
   text-decoration: none;
 }
 
-* {
```

Comment:
so we actually have this to showcase a common thing that apps do
if we have styles which are affected by this reset, then we need to fix our styles to be more resilient to this particular reset

Reply:
Sounds good to me. We probably have a few other components that are also affected too.